cpp/src/arrow/flight/sql/server.cc

```diff
@@ -795,7 +795,7 @@ std::shared_ptr<Schema> SqlSchema::GetTypeInfoSchema() {
       field("column_size", int32()),
       field("literal_prefix", utf8()),
       field("literal_suffix", utf8()),
-      field("create_params", list(utf8()), false),
+      field("create_params", list(field("item", utf8(), false)), false),
```

Comment:
Isn't the field itself nullable? It's a nullable list of non-null strings?

Reply:
I think I confused myself. The correct should be a List<utf8 not null>. I fixed it